### PR TITLE
add Entwine recipe

### PR DESCRIPTION
**Pull request by hobu**
Start working on an Entwine recipe. It is based on a branch for now. We will keep updating our branch for Entwine to incorporate any items needed to get Conda happy before providing a real release.

### CODE REVIEW

**Comment by scopatz** [comment]
So I don't think that you need the activate/deactive scripts, since they are empty.  Please remove them. Thanks for putting this in @hobu!

**Comment by hobu** [comment]
@scopatz I stubbed the scripts because I think we will need to do something in the near future before the actual release. If we don't actually need them, I will remove.

**Comment by scopatz** [comment]
Alright, sounds good!

**Comment by marcelotrevisani** [comment]
Should we merge this @scopatz ?

**Comment by marcelotrevisani** [comment]
I will just ping @hobu   :)

**Comment by hobu** [comment]
We are waiting for the Entwine 2.0 release to happen before merging this if possible. It can be closed/reopened if that's what you prefer. I just wanted to get it working as release preparation. When @connormanning issues the Entwine 2.0 release, I will make some more activity here.

Thanks for your patience.

**Comment by marcelotrevisani** [comment]
Hi @hobu , no problem at all!

Just to see if it is still in your plans. :)

**Comment by hobu** [comment]
Entwine 2.0.0rc1 was tagged, and it depends on the pdal 1.8.0 release. I have been having some trouble getting pdal 1.8.0 to properly pin GDAL, however.

**Comment by hobu** [comment]
How can I force the use of a newer compiler on circle? It is using gcc 4.8.2, which is likely much too old. You can see the crash during the test at the end of https://circleci.com/gh/conda-forge/staged-recipes/31614?

**Comment by hobu** [comment]
This recipe is ready to merge except for the spurious gcc 4.8.2 circle issue.

**Comment by hobu** [comment]
> This recipe is ready to merge except for the spurious gcc 4.8.2 circle issue.

I have tried to replicate this by building using the `condaforge/linux-anvil` container, but it builds with GCC 7+ when I use that, not the RH devtools gcc 4.8.2 that circle is complaining about. I'm at a loss as to what is going on or how to fix this.

**Comment by hobu** [comment]
@marcelotrevisani Sorry to ping, but can you provide some guidance here?

**Comment by scopatz** [comment]
@hobu - see https://conda-forge.org/docs/testing.html?#docker-tests

**Comment by hobu** [comment]
According to Marious van Niekerk on gitter, staged-recipes doesn't run the full compiler matrix. Even though the recipe specifies `{{ compiler('cxx')}}`, it is getting the container default GCC 4.8.2 rather than GCC 7.x. This is much too old for the software. I would like to propose to graduate this recipe, and I would continue to work through any compiler issues there should there be any.

**Comment by chambbj** [comment]
If I understand correctly, https://github.com/conda-forge/staged-recipes/pull/7116 was supposed to improve this situation with gcc7 support. There was no apparent change yesterday, but perhaps the PR just hadn't fully trickled its way into operation yet. Attempting to rerender and rebuild to see if it makes a difference.

@conda-forge-admin please rerender

**Comment by scopatz** [comment]
@hobu - BTW rerendering won't work in the staged-recipes repo because it is not a feedstock.

**Comment by hobu** [comment]
As mentioned up thread, this recipe is ready to go except for the need for the newer compiler which does not seem to be available on staged-recipes.

**Comment by hobu** [comment]
@scopatz This is green now. Can we get a feedstock and merge? Thanks so much.

File: .circleci/checkout_merge_commit.sh [hunk 1]
```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
 
 
-# Add `master` branch to the update list.
-# Otherwise CircleCI will give us a cached one.
```
Comment (scopatz) [inline .circleci/checkout_merge_commit.sh]: Please don't make changes outside of the recipes dir in a recipe PR. Please put this in as a separate PR. Thanks!

Comment (hobu) [reply .circleci/checkout_merge_commit.sh]: I'll make a new PR.
